src/vellum/workflows/nodes/core/retry_node/node.py

```diff
@@ -22,13 +23,15 @@ class RetryNode(BaseAdornmentNode[StateType], Generic[StateType]):
 
     max_attempts: int - The maximum number of attempts to retry the Subworkflow
     delay: float = None - The number of seconds to wait between retries
+    timeout: float = None - The number of seconds to wait for the Subworkflow to complete
```

Comment:
agh I reread the original slack thread , and I should've interjected earlier. the `timeout` setting referenced by Noa was surrounding this prompt timeout: https://github.com/vellum-ai/vellum/blob/2fa6eca28683fbcccb5d63de66059d5157baee38/django/app/domain_models/types/ml_model.py#L240

Strictly speaking, we may not need this attribute, the original ticket was more focused on adding delay to the default retry node attributes. We could finish SDK support through this PR, but let's defer any serialization/codegen support to focus on the higher urgency items we have for this week